snuba/datasets/entities/generic_metrics.py

```diff
@@ -13,51 +14,157 @@
     SubscriptableMapper,
 )
 from snuba.clickhouse.translators.snuba.mapping import TranslationMappers
+from snuba.datasets.entities.metrics import (
+    AggregateCurriedFunctionMapper,
+    AggregateFunctionMapper,
+)
```

Comment:
not sure if we consider this bad practice -- these couldn't directly go into the standard mappers file because of mypy issues but I didn't spend a lot of time trying to figure out the root cause.